src/views/Media/NewsletterSignup.vue

```diff
@@ -0,0 +1,233 @@
+<template>
+  <div id="cve-secondary-page-main-container">
+    <div class="columns is-centered">
+      <div class="column is-12-desktop is-12-tablet cve-main-column-content-width">
+        <main id="cve-main-page-content" role="main">
+          <div class="content">
+            <h1 class="title">Email Newsletter</h1>
+            <h2 class="title">Subscribe to the CVE<sup>®</sup> Program Newsletter</h2>
+            <p class="content">
+              <span class="has-text-weight-bold">CVE-Announce</span> provides general news about CVE such as new CNAs, new
+              website features, CVE in the news, etc. Messages are sent infrequently, once a
+              month or less.
+            </p>
+            <div v-if="!submitted" class="cve-white-bg-gray-border-container">
+              <form id="emailSignup" class="cve-top-margin" @submit="checkForm" novalidate="true">
```

Comment:
1. replace `@submit="checkForm"` with `@submit.prevent="checkForm"` to follow Vue's [event modifier convention](https://vuejs.org/v2/guide/events.html#Event-Modifiers)
2. Making change in item 1 makes line 210 unnecessary; please remove `e.preventDefault();`

Reply:
Done